src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { User } from '../User';
+
+export const Todo = ({ title, completed, user }) => (
+  <>
+    <strong>Todo: </strong>
+    {`${title} `}
+    <strong>Status: </strong>
+    {`${completed} `}
+    <strong>Name: </strong>
+    <User {...user} />
```

Comment:
if you need the name only, is better to pass only the name
```suggestion
    <User name={user.name} />
```